BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -137,7 +132,7 @@ - (void) setupWithConfiguration:(BlueShiftConfig *)config {
             [self fetchInAppNotificationFromDB];
         } failure:^(NSError *error){ }];
     }
-    //Mark app open if device token is already present, else delay it till app receves device token
+    //Mark app open if device token is already present, else delay it till app receves device token and fires identify
```

Comment:
maybe add a space between the `/` and `Mark`

Reply:
Fixed